src/backend/aspen/api/conftest.py

```diff
@@ -55,6 +55,21 @@ async def async_db() -> AsyncGenerator[AsyncPostgresDatabase, None]:
         f"grant all privileges on database {database_name} to {USERNAME}"
     )
 
+    # Install extensions we use to the test database.
+    ext_query = await admin_session.execute("SELECT extname FROM pg_extension")
+    extensions = [row["extname"] for row in ext_query]
+    test_admin_uri = (
+        f"postgresql+asyncpg://postgres:password_postgres@database:5432/{database_name}"
+    )
```

Comment:
it would be better to generate the admin uri based on some config (or better yet, figure out how to grant more access to our test user!) but we can sort that out later if this is getting out of hand.

Reply:
We can give the test user superuser privileges, but that seemed like a bad idea to me 🥴

Reply:
Wait - I take this whole comment back - we're already doing stupid hardcoded db uri's in this method. We don't have to make this smarter right now.